### ISSUE TITLE
Rename plugin unit tests to integration tests?

### ISSUE THREAD

**Issue by dingo-d** [wp-cli/handbook#247]
The title of the chapter `Plugin Unit Tests` is a bit misleading. 

Unit tests are usually done when adding/modifying some core feature/component, and plugins work within WordPress so the tests in the plugin are in fact integration tests.

The chapter title should be renamed to `Plugin Integration Tests`.

**Comment by schlessera** [comment]
I agree, the entire scaffolding feature of tacking the WP Core unit tests onto plugins has lead to a lot of confusion.

I have a few things I want to change with the scaffolding in the near future, and will certainly address this as well.

**Comment by spicecadet** [comment]
https://github.com/wp-cli/handbook/pull/357

There is another mention of Unit tests at the bottom. The file name has unit rather than integration. Should these change as well?
